permissions/helpers.py

```diff
@@ -119,6 +134,10 @@ def can_modify_application(user: AnyUser, application: Application) -> bool:
         return True
     if application.application_round.service_sector is None:
         return False
+    if has_general_permission(user, GeneralPermissionChoices.CAN_ALLOCATE_APPLICATIONS):
+        return True
+    if has_general_permission(user, GeneralPermissionChoices.CAN_VALIDATE_APPLICATIONS):
```

Comment:
replace with check for **Unit**PermissionChoices.CAN_**HANDLE**_APPLICATIONS (as in `can_read_application`)

Reply:
Done.